cross-chain/arbitrum/deploy_l2/02_deploy_arbitrum_wormhole_gateway.ts

```diff
@@ -0,0 +1,74 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { log } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // These are the fake random addresses for local development purposes only.
+  const fakeTokenBridge = "0x0af5DC16568EFF2d480a43A77E6C409e497FcFb9"
+  const fakeWormholeTBTC = "0xe1F0b28a3518cCeC430d0d86Ea1725e6256b0296"
+
+  const L2TokenBridge = await deployments.getOrNull("TokenBridge") // L2
+  const WormholeTBTC = await deployments.getOrNull("WormholeTBTC") // L2
+  const ArbitrumTBTC = await deployments.get("ArbitrumTBTC")
+
+  let tokenBridgeAddress = ""
+  if (L2TokenBridge && helpers.address.isValid(L2TokenBridge.address)) {
+    log(`using existing L2 TokenBridge at ${L2TokenBridge.address}`)
+    tokenBridgeAddress = L2TokenBridge.address
+  } else if (hre.network.name === "hardhat") {
+    log(`using fake L2 TokenBridge at ${fakeTokenBridge}`)
+    tokenBridgeAddress = fakeTokenBridge
+  } else {
+    throw new Error("deployed L2TokenBridge contract not found")
+  }
+
+  let wormholeTBTCAddress = ""
+  if (WormholeTBTC && helpers.address.isValid(WormholeTBTC.address)) {
+    log(`using existing L2 WormholeTBTC at ${WormholeTBTC.address}`)
+    wormholeTBTCAddress = WormholeTBTC.address
+  } else if (hre.network.name === "hardhat") {
+    log(`using fake L2 WormholeTBTC at ${fakeWormholeTBTC}`)
+    wormholeTBTCAddress = fakeWormholeTBTC
+  } else {
+    throw new Error("deployed L2TokenBridge contract not found")
+  }
+
+  const [, proxyDeployment] = await helpers.upgrades.deployProxy(
+    "ArbitrumWormholeGateway",
+    {
+      contractName: "L2WormholeGateway",
```

Comment:
Being specific here may help us reduce confusion.

```suggestion
      contractName: "@keep-network/tbtc-v2/contracts/l2/L2WormholeGateway.sol:L2WormholeGateway",
```

Reply:
[d2c5a17](https://github.com/keep-network/tbtc-v2/pull/554/commits/d2c5a174919fabe37252673cc75249341c06e1fb)